### DISCUSSION TITLE
Toggle number sign at any cursor position

### DISCUSSION THREAD

**Discussion by beroso** [dm4t2/vue-currency-input#380] [Ideas]
Currently it only accepts minus sign at first position, and if we need to change the number to positive we need to erase the sign.
I think it will be practical to toggle number sign by just pressing `-` at any cursor position.

**Comment by DOUGLASMD1** [comment]
Olá @beroso 

Me inspirei nesse trecho para ter esse comportamento: https://github.com/phiny1/v-currency-field/blob/2eba8409be3fb53953571047a15560d097afea17/src/VCurrencyField.vue#LL139C31-L139C31

E funcionou :100:
